common/utils/util.js

```diff
@@ -155,14 +149,15 @@ const processReplacements = (input, replacementDetails) => {
 
 const formatDateWith = pattern => isoString => {
   const parsedDate = parseISO(isoString)
-  return isValid(parsedDate) ? format(parsedDate, pattern) : null
+  return isValid(parsedDate) ? formatInTimeZone(parsedDate, 'Europe/London', pattern) : null
 }
 
 const prettyDate = formatDateWith('do MMMM y')
 const prettyDateAndTime = formatDateWith('eeee do MMMM y H:mm')
 
-const ageFrom = (dateOfBirth, today = new Date()) => {
-  const parsedDate = parseISO(dateOfBirth)
+const ageFrom = (dateOfBirth, now = new Date()) => {
+  const today = startOfDay(utcToZonedTime(now, 'Europe/London'))
```

Comment:
I've added the additional `startOfDay` here as technically we were calculating the age on the literal years elapsed, so in theory they could have increased in age mid-way through the day. It's an edge case, but fixed now at least.